src/diffusers/models/modeling_utils.py

```diff
@@ -859,6 +970,47 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
 
         return model
 
+    @wraps(torch.nn.Module.cuda)
```

Comment:
These methods exist to align with transformers, right? If so, could be worth adding a comment.

Reply:
Update: done in 44c410996.